docs/iot/iot-partners/thingspark.md

```diff
@@ -0,0 +1,27 @@
+# Tizen IoT with thingspark Cloud
+
+The thingspark service is a cloud-based IoT solution that can be connected with various things. 
+
+Tizen developers can easily develop applications that send IoT data to the cloud in real-time.
```

Comment:
@xbldx 
Request you to add "can" before "send IoT..."
Tizen developers can easily develop applications that can send IoT data to the cloud in real-time.